src/views/SystemProfile/components/SystemDetails/index.tsx

```diff
@@ -25,432 +24,394 @@ import Divider from 'components/shared/Divider';
 import SectionWrapper from 'components/shared/SectionWrapper';
 import Tag from 'components/shared/Tag';
 import useCheckResponsiveScreen from 'hooks/checkMobile';
+import GetSystemProfileDetailsQuery from 'queries/GetSystemProfileDetailsQuery';
 import {
-  tempCedarSystemProps,
-  tempLocationProp
-} from 'views/Sandbox/mockSystemData';
+  GetSystemProfileDetails,
+  GetSystemProfileDetailsVariables
+} from 'queries/types/GetSystemProfileDetails';
+import { SystemProfileSubviewProps } from 'types/systemProfile';
+import NotFound from 'views/NotFound';
+import { showVal } from 'views/SystemProfile';
 
-// import { GetCedarSystems_cedarSystems as CedarSystemProps } from 'queries/types/GetCedarSystems';
 import 'index.scss';
 
-type SystemDetailsProps = {
-  system: tempCedarSystemProps; // TODO: Once additional CEDAR data is define, change to GQL generated type
-};
-
-// Function for determining if a system has any URLs (otherwise results in alert)
-const checkURLsExist = (locations: tempLocationProp[]): boolean => {
-  return locations.some((location: tempLocationProp) => location.url);
-};
-
-const SystemDetails = ({ system }: SystemDetailsProps) => {
+const SystemDetails = ({ system }: SystemProfileSubviewProps) => {
   const { t } = useTranslation('systemProfile');
   const isMobile = useCheckResponsiveScreen('tablet');
   const flags = useFlags();
+  const { loading, error, data } = useQuery<
+    GetSystemProfileDetails,
+    GetSystemProfileDetailsVariables
+  >(GetSystemProfileDetailsQuery, {
+    variables: {
+      cedarSystemId: system.id
+    }
+  });
```

Comment:
Same comment as the ATO section here - Could this be included in the `system` param by modifying `GetSystemProfileQuery.ts` to include this?

Reply:
I just merged all those into the same query at the parent level, and was able to remove the other queries pretty seamlessly.   It removes the loading time of the subpages